torchgeo/trainers/segmentation.py

```diff
@@ -122,7 +122,22 @@ def configure_losses(self) -> None:
             )
 
     def configure_metrics(self) -> None:
-        """Initialize the performance metrics."""
+        """Initialize the performance metrics.
+
+        * Multiclass Pixel Accuracy: Ratio of correctly classified pixels.
+        * Multiclass Jaccard Index (IoU): Per-pixel overlap between predicted and
+          actual segments.
```

Comment:
Might also be worth documenting that higher is better. This isn't true for all of our metrics.

Reply:
Yeah, that would be helpful for quick reference and those unfamiliar with the metrics. Maybe something like `{Higher/Lower} values indicate better agreement.`?

Reply:
I would be fine with that or even something as simple as `{Higher/Lower} values are better.`

Reply:
Sounds good. I'll add that in.